docs/enable-kerberos.md

```diff
@@ -75,15 +76,16 @@ kadmind
 
 ### 2.1 Add smartserver Kerberos principal to database and export it to keytab.
 ```
-kadmin.local:addprinc -randkey smartserver/{hostname}
-kadmin.local:xst -k /xxx/xxx/smartserver-{hostname}.keytab smartserver/{hostname}
+kadmin.local:addprinc -randkey {username}/{hostname}
+kadmin.local:xst -k /xxx/xxx/smartserver-{hostname}.keytab {username}/{hostname}
 ```
-Note: replace the hostname with the hostname of the smart server
+**Note:** replace the username with the HDFS user who has the correct permission to execute actions and replace hostname with hostname of the smart server
```

Comment:
I am not sure what "HDFS user" means. Is it the user configured in hadoop-env.sh? I forget why we used smartserver or smartagent in previous tests.

Reply:
Can the username for smart agent and smart server be the same one here?

Reply:
The user performing the operation changed to smartagent or smartserver after completing kerberos configuration. But these two users do not have permission to operate on some specific files. “HDFS user” means it is best to have the highest operating authority for HDFS like the user configured in dfs.permissions.superusergroup.  
![image](https://user-images.githubusercontent.com/32612402/76483184-4b7f8b80-6451-11ea-9029-2e50e99dbaf9.png)

Reply:
Of coure, we can set this "HDFS user" as other users name. But we should give the user proper authority by Ranger or  other authorization management application.

Reply:
OK, please also use {username} to replace "smartagent"/"smartserver" for setting the value of _smart.server.kerberos.principal/smart.agent.kerberos.principal_ in the below configuration template of smart-site.xml.

Reply:
i.e., line 119, 127

Reply:
Thanks. I have replaced "smartagent"/"smartserver" in the smart-site.xml.